tests/scenarios/rest_text/run.sh

```diff
@@ -0,0 +1,116 @@
+    returned=$(curl -s -k https://user1:${TOKEN}@localhost:50443/databases/db1/tables/table1/rows/1 | awk -F "CTEXT\":\"" '{print $2}' |  awk -F "\"\}" '{print $1}')
+    if [[ "${returned}" != "${test_value_out_rest}" ]]; then
+        _log "ERROR" "SIOCLI | Expected: >"${test_value_out_rest}"< and got >"${returned}"<"
+        _failExit
+    fi
+    _log "INFO" "SIOCLI(SUCCESS) | Expected: >"${test_value_out_rest}"< and got >"${returned}"<"
+}
+
+
+function test_value_from_rest () {
+    test_value_in_rest="${1}"
+    test_value_out_rest="${2}"
+    _log "INFO" "REST put: >"${test_value_in_rest}"<"
+    curl -s -o /dev/null -X PUT -d "[{\"ctext\":\"${test_value_in_rest}\"}]" -k https://user1:${TOKEN}@localhost:50443/databases/db1/tables/table1/rows/1
+    returned=$(curl -s -k https://user1:${TOKEN}@localhost:50443/databases/db1/tables/table1/rows/1 | awk -F "CTEXT\":\"" '{print $2}' |  awk -F "\"\}" '{print $1}')
+    if [[ "${returned}" != "${test_value_out_rest}" ]]; then
+        _log "ERROR" "REST | Expected: >"${test_value_out_rest}"< and got >"${returned}"<"
+        _failExit
+    fi
+    _log "INFO" "REST(SUCCESS) | Expected: >"${test_value_out_rest}"< and got >"${returned}"<"
+}
+
+## Specific test parameters
+
+## =============================================
+## TEST HEADER
+## =============================================
+_TestBegin
+_Prepare
+_StartSiodb
+
+# ## =============================================
+# ## TEST
+# ## =============================================
+
+
+_log "INFO" "Creating test objects"
+_RunSqlScript "${SCRIPT_DIR}/create_objects.sql" 90
+TOKEN=$(openssl rand -hex 64)
+_RunSql "create user user1"
+_RunSql "alter user user1 add token token1 x'${TOKEN}'"
+
+# # Post initial chars
+_RunSql "insert into db1.table1 (ctext) values (' ')"
+
+# test value
+test_value_from_rest '' ''
+test_value_from_siocli '' ''
+test_value_from_rest ' ' ' '
+test_value_from_siocli ' ' ' '
+
+# Back Slash
+test_value_from_siocli '\' '\\'
+test_value_from_rest '\\' '\\'
+test_value_from_siocli '-\-' '-\\-'
+test_value_from_rest '-\\-' '-\\-'
+test_value_from_siocli '\-' '\\-'
+test_value_from_rest '\\-' '\\-'
+test_value_from_siocli '-\' '-\\'
+test_value_from_rest '-\\' '-\\'
+
+
+# Double Quote
+test_value_from_siocli '"' '\"'
+test_value_from_rest '\"' '\"'
+test_value_from_siocli '\"' '\\\"'
+test_value_from_rest '\\\"' '\\\"'
+test_value_from_siocli '""""""""""' '\"\"\"\"\"\"\"\"\"\"'
+test_value_from_rest '\"\"\"\"\"\"\"\"\"\"' '\"\"\"\"\"\"\"\"\"\"'
+
+# Slash
+test_value_from_rest '\/' '\/'
+test_value_from_rest '/' '\/'
+
+# Escaped chars
+test_value_from_siocli '
+' '\n'
+test_value_from_rest '\n' '\n'
+test_value_from_siocli '\b' '\\b'
+test_value_from_rest '\b' '\b'
+test_value_from_siocli '\f' '\\f'
+test_value_from_rest '\f' '\f'
+test_value_from_siocli '\r' '\\r'
+test_value_from_rest '\r' '\r'
+test_value_from_siocli '\t' '\\t'
+test_value_from_rest '\t' '\t'
```

Comment:
missing test for the `'\v'`

Reply:
@cxxman  I'm not sure '\v' is part of the JSON specification. Isn't it?

Reply:
Speaking generally, what we do here, even after such fixes, still not completely correct, because we would have generated byte-per-byte encoding for valid multibyte UTF-8 sequences, which is not correct, because they must be either one or two `\uXXXX` (see [RFC 7159](https://tools.ietf.org/html/rfc7159) Section 7). But I suggest to do this as a separate fix.